### DISCUSSION TITLE
如何配置

### DISCUSSION THREAD

**Discussion by Kadxy** [wzunjh/Online-learning-platform#5] [Q&A] [answered]
我想了解一下具体的步骤，你能详细告诉我如何进行配置吗？我需要知道每一步的操作以及需要注意的问题，这样我才能确保我在进行配置的时候不会出现错误。如果有任何需要特别注意或者需要提前准备的事项，也请你一并告诉我。

**Comment by JoeNJH** [comment, marked as answer]
私聊我
